docs/python/tutorial-django.md

```diff
@@ -380,12 +386,7 @@ In this section you start by creating a single page using a template. In subsequ
         content = "<strong>Hello there, " + name + "!</strong> It's " + formatted_now
 
         return render(
-            request,
-            'hello/hello_there.html',
-            {
-                'title': 'Hello, Django',
-                'content': content,
-            }
+            request, "hello/hello_there.html", {"title": "Hello, Django", "content": content}
```

Comment:
Revert this change. I want to keep the template parameters spelled out and highly visible the first time through. Later it's OK to condense, but at this point the purpose is to help people understand what they're looking at, and condensing it just makes it harder to parse.